.github/workflows/publish-docs.yml

```diff
@@ -4,23 +4,10 @@ on:
   schedule:
     - cron: "0 0 * * 1-5"
   workflow_dispatch:
-    inputs:
-      live-run:
-        type: boolean
-        description: Live-run
-        required: false
-      version:
-        type: string
-        description: Release number
-        required: false
-      zenoh-version:
-        type: string
-        description: Release number of Zenoh
-        required: false
-      branch:
-        type: string
-        description: Release branch
-        required: false
+  push:
+    tags:
+      - v[0-9]+.*
```

Comment:
We don't use the `v` character before the version tag

Reply:
Removed the `v`